instructor/process_response.py

```diff
@@ -249,6 +250,15 @@ def handle_response_model(
             Mode.warn_mode_functions_deprecation()
             new_kwargs["functions"] = [response_model.openai_schema]
             new_kwargs["function_call"] = {"name": response_model.openai_schema["name"]}
+        elif mode in {Mode.STRUCTURED_OUTPUTS}:
```

Comment:
why not call it tools_strict 

since it looks like we're not using the parse method?

Reply:
That's a good point, let me rename the method to TOOLS_STRICT then